web/models.py

```
@@ -21,6 +23,9 @@
 from django.utils.translation import gettext_lazy as _
 from markdownx.models import MarkdownxField
 from PIL import Image
+from django.contrib.admin import ModelAdmin, register
+from django.contrib.admin.views.main import ChangeList
```

Ruff (0.8.2)

`django.contrib.admin.views.main.ChangeList` imported but unused

Remove unused import: `django.contrib.admin.views.main.ChangeList`

(F401)